llvm/test/tools/llvm-objcopy/ELF/decompress-sections.test

```diff
@@ -4,13 +4,42 @@
 # RUN: yaml2obj %s -o %t
 # RUN: llvm-objcopy --decompress-debug-sections %t %t.de
 # RUN: llvm-readelf -S %t.de | FileCheck %s
+# RUN: llvm-objcopy --compress-sections '*nonalloc=none' --compress-sections .debugx=none %t %t.de
```

Comment:
These should use different output file names, to make it easier to do post mortems on the test.